### ISSUE TITLE
How do I add eslint-plugin-react to the CLIEngine

### ISSUE THREAD

**Issue by muuvmuuv** [yannickcr/eslint-plugin-react#1959]
I wondering how I can add this plugin to the CLIEngine. I looked in the docs and it says I need to pass a plugin object but I do not know what does that mean.

That is what I've so far:

```javascript
import * as ESLintPluginReact from "eslint-plugin-react";
const cli: CLIEngine = new CLIEngine(cliOptions);
cli.addPlugin("eslint-plugin-react", ESLintPluginReact);
```

**Comment by alexzherdev** [comment]
Not sure how this would look like in typescript, but in javascript I _think_ it should be
```js
import ESLintPluginReact from "eslint-plugin-react";
```

**Comment by ljharb** [comment]
`import * as` from CJS Modules is TS’ broken module system; you’d have to enable synthetic imports to get it to work properly.

**Comment by muuvmuuv** [comment]
Fixed this by using the build in dependencies manager from the system I wanted to create a plugin for. But thanks for the fast response!

From the programm [Unibeautify](https://github.com/unibeautify/unibeautify)
```
const ESLintPluginReact = dependencies.get<NodeDependency>(
        "ESLintPluginReact"
      ).package;
```

@ljharb I'll keep that in mind for other projects!
